web/.eslintignore

```diff
@@ -0,0 +1 @@
+/src/services/thirdparty/**
```

Comment:
not sure why you want to ignore this? We wrote that code :D

Reply:
It contained `mixpanel.js`, which had some minified code. So I decided to ignore the whole folder.
Better to only ignore mixpanel.js?

Reply:
Removed the file.